DANS-KNAW/easy-pid-generator
# Refactored from war-project to jar-project

#### When applied it will
- Not change the interface of the `easy-pid-generator`
- Internally the project is now a `jar`-project
#### Where should the reviewer @DANS-KNAW/easy start?
#### How should this be manually tested?
- Deploy on deasy an verify that you can still create PIDs the usual way: `curl -X POST http://localhost:8082/pids?type=urn`
- Verify that the service can still be started and stopped as usual: `service easy-pid-generator start|stop|status`
#### related pull requests on github
